lib/tasks/data.rake

```diff
@@ -1,5 +1,6 @@
 DATA_MIGRATION_SERVICES = [
   # do not delete or edit this line - services added below by generator
+  'DataMigrations::BackfillWithdrawnOrDeclinedForCandidateByProvider',
```

Comment:
Are data migrations guaranteed to run after database migrations?

Reply:
Yes. Data migrations are configured to run in an `enhance` block which is executed after the specified rake task.

https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/lib/tasks/data.rake#L44

More info: https://stackoverflow.com/questions/35874930/run-one-rake-task-after-another-completes